CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+* Add go1.22.3
+* Add go1.21.10
+* go1.22 defaults to go1.22.3
+* go1.21 defaults to go1.21.10
 * Add support for heroku-24
 * Drop support for installing bzr-hosted dependencies directly; bzr-hosted
 
```

Comment:
Looking at the rendered view, the previous v190 release heading is a broken link:
https://github.com/heroku/heroku-buildpack-go/blob/main/CHANGELOG.md#v190---2024-04-05

It looks like the v190 release prep PR was manually created (as part of adding support for the new versions), so didn't have the footer half of the link created that would normally be added by the prepare release workflow:
https://github.com/heroku/heroku-buildpack-go/blob/c3ecc34423ad7403b0014964ca9b1cdf8cf5562a/CHANGELOG.md?plain=1#L968-L969
https://github.com/heroku/heroku-buildpack-go/blob/main/.github/workflows/prepare-release.yml

Would you mind fixing that now? :-)

Reply:
Opened a PR for the fix: https://github.com/heroku/heroku-buildpack-go/pull/555.